_episodes/01-short-introduction-to-Python.md

```diff
@@ -73,6 +88,8 @@ Hello World
 ~~~
 {: .output}
 
+For the remainder of this episode, use your chosen text editor when you create a script.
```

Comment:
```suggestion
For the remainder of this lesson, you could use your chosen text editor to write example commands and your solutions to exercises as scripts.
```

I think this should be worded more as a suggestion than an instruction, because it is not clear how learners will be asked to follow along with the lesson in a workshop (e.g. Instructors may be using Jupyter, or an IDE, the REPL prompt in a terminal, etc).